db/migrate/20150513162113_remove_admin_users_departments_join_table.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAdminUsersDepartmentsJoinTable < ActiveRecord::Migration
+  def change
+    drop_table 'admin_users_departments'
```

Comment:
Oh, I thought I knew all the secret migration helpers.  That's a new one to me.  Thanks!